packages/common/src/groups/addGroupMembers.ts

```diff
@@ -0,0 +1,83 @@
+import { IAuthenticationManager } from "@esri/arcgis-rest-request";
+import { IUser } from "@esri/arcgis-rest-types";
+import { failSafe } from "../utils";
+import { autoAddUsers } from "./autoAddUsers";
+import { inviteUsers } from "./inviteUsers";
+import { IAddGroupMembersResult, IAddOrInviteMemberResponse } from "./types";
+
+/**
+ * Add or invite N users to a single group.
+ * if autoAdd is true (if the user doing the adding has the correct priv as determined outsides of this function)
```

Comment:
Please link to the fn thata determines if the user can autoAdd vs just invite and/or explain those conditions. Also fix casing and spelling :)